docs/data.md

```diff
@@ -1,40 +1,59 @@
+Data referred to on this page is available under various licenses, and the onus is on the user to understand the conditions of those licenses.
 
-Global model data is also available from the NOAA Global Forecast System (GFS). See https://www.ncei.noaa.gov/products/weather-climate-models/global-forecast for more information.
+For addtional information about downloading and preparing sample data, see [this tutorial](project:./tutorials/First_Data_Fetching.md).
 
-## CliMetLab
-ECMWF has developed the CliMetLab python package to simplify access to a large range of climate and meteorological datasets. See https://climetlab.readthedocs.io/en/latest/
+## Weather and Climate Data
 
-## NOAA ISD dataset
-The NOAA Integrated Surface Database provides hourly point-based (aka in-situ) data globally and is a good starting point for understanding how to work with point-based data. Point-based observations are shared routinely between countries for the purposes of weather modelling.
+### Datasets
 
-See https://www.ncei.noaa.gov/products/land-based-station/integrated-surface-database for more information.
+#### Gridded Global Numerical Weather Prediction Data
 
-## Gridded model reanalysis data
-Reanalysis data is useful for providing a very long history of atmospheric conditions. The ERA5 dataset is a well known and widely used global reanalysis dataset.
+Global numerical weather prediction (NWP) models are used to generate medium range forecasts and provide the initial and boundary conditions for higher-resolution regional models. Their global coverage makes them a good starting point for demonstrating the application of scoring methods in any region of interest.
 
-For more information see https://www.ecmwf.int/en/forecasts/datasets/reanalysis-datasets/era5 and
-https://github.com/pangeo-data/WeatherBench
+The Bureau of Meteorology provides global model data from the ACCESS NWP system. See [https://doi.org/10.25914/608a993391647](https://doi.org/10.25914/608a993391647) for more information.
 
-## Gridded satellite (observation) data
-Satellite data varies according to the type of orbit and purpose of the mission. It is too complex to quickly address in a demonstration. A guide on working with satellite data may be added in future.
+Global model data is also available from the National Oceanic and Atmospheric Administration (NOAA) Global Forecast System (GFS). See [https://www.ncei.noaa.gov/products/weather-climate-models/global-forecast](https://www.ncei.noaa.gov/products/weather-climate-models/global-forecast) for more information.
 
-## Gridded radar (observation) data
-Radar data also varies according to region and is not a globally standardised data set. Information on Australian based radars can be found at https://www.openradar.io/
+#### Point-Based Data
+
+Point-based observations are shared routinely between countries for the purposes of weather modelling.
+
```

Comment:
I couldn't work out whether this sentence should be retained, and if so, where in the section it should go